docs/sources/datahub-cloud-event-source.md

```diff
@@ -0,0 +1,114 @@
+fail to be processed will simply be logged to a `failed_events.log` file for further investigation (dead letter queue). The DataHub Cloud Event Source will continue to make progress against the underlying topics and continue to commit offsets even in the case of failed messages. 
+
+If you've configured your Action pipeline `failure_mode` to be `THROW`, then events which fail to be processed result in an Action Pipeline error. This in turn terminates the pipeline before committing offsets back to DataHub Cloud. Thus the message will not be marked as "processed" by the Action consumer.
+
+## Supported Events
+
+The DataHub Cloud Event Source produces
+
+- [Entity Change Event V1](../events/entity-change-event.md)
+
+Note that the DataHub Cloud Event Source does _not_ yet support the full [Metadata Change Log V1](../events/metadata-change-log-event.md) event stream. 
+
+## Configure the Event Source
+
+Use the following config(s) to get started with the DataHub Cloud Event Source. 
+
+### Quickstart
+
+To start listening for new events from now, you can use the following recipe: 
+
+```yml
+name: "unique-pipeline-name"
+datahub:
+  server: "https://<your-organization>.acryl.io"
+  token: "<your-datahub-cloud-token>"
+source:
+  type: "datahub-cloud"
+action:
+  # action configs
+```
+
+Note that the `datahub` configuration block is **required** to connect to your DataHub Cloud instance. 
+
+### Advanced Configurations 
+
+To reset the offsets for the action pipeline and start consuming events from 7 days ago, you can use the following recipe: 
+
+```yml
+name: "unique-pipeline-name"
+datahub:
+  server: "https://<your-organization>.acryl.io"
+  token: "<your-datahub-cloud-token>"
+source:
+  type: "datahub-cloud"
+  config:
+    lookback_days: 7                           # Look back 7 days for events
+    reset_offsets: true                        # Ignore stored offsets and start fresh
+    kill_after_idle_timeout: true              # Enable shutdown after idle period
+    idle_timeout_duration_seconds: 60          # Idle timeout set to 60 seconds
+    event_processing_time_max_duration_seconds: 45  # Max processing time of 45 seconds per batch
+action:
+  # action configs
+```
+
+Note that the `datahub` configuration block is **required** to connect to your DataHub Cloud instance. 
+
+<details>
+  <summary>View All Configuration Options</summary>
+  
+  | Field                                 | Required | Default                       | Description                                                                               |
+  | ------------------------------------- | :------: | :---------------------------: | ----------------------------------------------------------------------------------------- |
+  | `token`                               |    ✅    | None                          | Authentication token if required for accessing the event source.                          |
```

Comment:
this is odd- isn't a token explicitly not required?

Reply:
it's not - good catch. need to remove this.